genesyscloud/resource_genesyscloud_flow.go

```diff
@@ -138,26 +135,32 @@ func isForceUnlockEnabled(d *schema.ResourceData) bool {
 }
 
 func createFlow(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	log.Printf("Creating flow")
+	return updateFlow(ctx, d, meta)
+}
+
+func updateFlow(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	sdkConfig := meta.(*ProviderMeta).ClientConfig
 	architectAPI := platformclientv2.NewArchitectApiWithConfig(sdkConfig)
 
-	log.Printf("Creating flow")
```

Comment:
We should have some kind of log here.  It should probably indicate we are updating a flow.